tests/secret-providers/aws_secretmanager/aws_secretmanager_test.go

```diff
@@ -0,0 +1,452 @@
+  cooldownPeriod:  10
+  minReplicaCount: {{.MinReplicaCount}}
+  maxReplicaCount: {{.MaxReplicaCount}}
+  triggers:
+  - type: postgresql
+    metadata:
+      targetQueryValue: "4"
+      activationTargetQueryValue: "5"
+      query: "SELECT CEIL(COUNT(*) / 5) FROM task_instance WHERE state='running' OR state='queued'"
+    authenticationRef:
+      name: {{.TriggerAuthenticationName}}
+`
+
+	postgreSQLStatefulSetTemplate = `apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: {{.PostgreSQLStatefulSetName}}
+  name: {{.PostgreSQLStatefulSetName}}
+  namespace: {{.TestNamespace}}
+spec:
+  replicas: 1
+  serviceName: {{.PostgreSQLStatefulSetName}}
+  selector:
+    matchLabels:
+      app: {{.PostgreSQLStatefulSetName}}
+  template:
+    metadata:
+      labels:
+        app: {{.PostgreSQLStatefulSetName}}
+    spec:
+      containers:
+      - image: postgres:10.5
+        name: postgresql
+        env:
+          - name: POSTGRES_USER
+            value: {{.PostgreSQLUsername}}
+          - name: POSTGRES_PASSWORD
+            value: {{.PostgreSQLPassword}}
+          - name: POSTGRES_DB
+            value: {{.PostgreSQLDatabase}}
+        ports:
+          - name: postgresql
+            protocol: TCP
+            containerPort: 5432
+`
+
+	postgreSQLServiceTemplate = `apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: {{.PostgreSQLStatefulSetName}}
+  name: {{.PostgreSQLStatefulSetName}}
+  namespace: {{.TestNamespace}}
+spec:
+  ports:
+  - port: 5432
+    protocol: TCP
+    targetPort: 5432
+  selector:
+    app: {{.PostgreSQLStatefulSetName}}
+  type: ClusterIP
+`
+
+	lowLevelRecordsJobTemplate = `apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: postgresql-insert-low-level-job
+  name: postgresql-insert-low-level-job
+  namespace: {{.TestNamespace}}
+spec:
+  template:
+    metadata:
+      labels:
+        app: postgresql-insert-low-level-job
+    spec:
+      containers:
+      - image: ghcr.io/kedacore/tests-postgresql
+        imagePullPolicy: Always
+        name: postgresql-processor-test
+        command:
+          - /app
+          - insert
+        env:
+          - name: TASK_INSTANCES_COUNT
+            value: "20"
+          - name: CONNECTION_STRING
+            valueFrom:
+              secretKeyRef:
+                name: {{.SecretName}}
+                key: postgresql_conn_str
+      restartPolicy: Never
+  backoffLimit: 4
+`
+
+	insertRecordsJobTemplate = `apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: postgresql-insert-job
+  name: postgresql-insert-job
+  namespace: {{.TestNamespace}}
+spec:
+  template:
+    metadata:
+      labels:
+        app: postgresql-insert-job
+    spec:
+      containers:
+      - image: ghcr.io/kedacore/tests-postgresql
+        imagePullPolicy: Always
+        name: postgresql-processor-test
+        command:
+          - /app
+          - insert
+        env:
+          - name: TASK_INSTANCES_COUNT
+            value: "10000"
```

Comment:
As we are scaling to 1 instead of 2, we have to reduce the "queue length"
```suggestion
            value: "1000"
```